accname/name/comp_label.html

```diff
@@ -100,9 +99,14 @@ <h2>HTML elements that support aria-label</h2>
 <aside aria-label="label" data-expectedlabel="label" data-testname="label valid on aside element" class="ex">x</aside>
 <blockquote aria-label="label" data-expectedlabel="label" data-testname="label valid on blockquote element" class="ex">x</blockquote>
 <button aria-label="label" data-expectedlabel="label" data-testname="label valid on button element" class="ex">x</button>
-<dd aria-label="label" data-expectedlabel="label" data-testname="label valid on dd element" class="ex">x</dd>
-<dl aria-label="label" data-expectedlabel="label" data-testname="label valid on dl element" class="ex">x</dl>
-<dt aria-label="label" data-expectedlabel="label" data-testname="label valid on dt element" class="ex">x</dt>
+<dl aria-label="label" data-expectedlabel="label" data-testname="label valid on dl element" class="ex">
+  <!-- dt/dd pending listitemkey and listitemvalue roles: https://github.com/w3c/aria/issues/1662 -->
+  <!-- Currently these map to `term` and `definition` for which name is prohibited. -->
+  <!-- Using aria-label here would be an authoring error, however there is no UA requirement to ignore the author provided role. -->
```

Comment:
Actually based on the discussion today, perhaps we need another `tentative` test to invalidate this label based on the AccName update that requires ignoring `nameFrom:author` on roles with name prohibited.

Reply:
I think things are too much in flux rn and we should do another round of test updates once we figure out to do with name prohibited. Right now, ACCNAME says to return the empty string for prohibited names, which is counter to the test. However, I don't think there is a consensus on what actually should happen yet.